### ISSUE TITLE
Feedback CRUD: Update feedback

### ISSUE THREAD

**Issue by Vurpas** [Vurpas/Second-Roll-Healthcare#10]
FLAGGED wontFix BECAUSE WE DONT NEED THIS FEATURE.

Create a BASIC method in both controller and service that updates the comment of a specific feedback.
NOTE: Issue #9 needs to be done before this can be started.

- [ ] Create a method in controller with endpoint.
- [ ] Send data to service class and create method for updating the comment in the database.
- [ ] test in postman.
